.ci_helpers/docker/setup-services.py

```diff
@@ -18,17 +27,44 @@
 def parse_args():
     parser = argparse.ArgumentParser(description="Setup services for testing")
     parser.add_argument("--deploy", action="store_true", help="Flag to setup docker services")
+    parser.add_argument(
+        "--no-pull",
+        action="store_true",
+        help="Optional flag to pull the latest images",
```

Comment:
I think the help message should actually be something like this: "Optional flag to skip pulling the latest images from dockerhub"

Reply:
Oops that was from my initial iteration. Thanks for catching that!

Reply:
I'm still new to PR "suggestions"! Thanks for doing that.